src/utilities/get.ts

```diff
@@ -0,0 +1,17 @@
+import {GeneralObject} from '../types';
+
+function get(obj: GeneralObject | undefined, keypath: string) {
+  if (obj == null) return undefined;
+
+  const keys = keypath.split('.');
+  let acc = obj;
```

Comment:
Nit: I find it challenging to decipher abbreviated variable names, but that's a personal preference.

Reply:
`accumulator`, which is the name used in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce).

(more tedious-than-fun-fact: because I got familiar with functional programming using ruby my brain defaults to calling this parameter "[memo](https://ruby-doc.org/core-2.1.0/Enumerable.html#method-i-reduce)")

Reply:
...which then creates confusion with `memoize`. Naming is hard.

Reply:
I would prefer `accumulator`, but that's just me.